.changeset/beige-schools-kneel.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/react": patch
```

Comment:
```suggestion
"@primer/react": minor
```

I think this would be a minor since it falls under [this case](https://github.com/primer/react/blob/main/contributor-docs/versioning.md#the-type-of-a-prop-is-broadened)